src/simsopt/field/coil.py

```diff
@@ -8,7 +8,7 @@
 import simsoptpp as sopp
 
 
-__all__ = ['Coil', 'Current', 'coils_via_symmetries',
+__all__ = ['Coil', 'Current', 'ScaledCurrent', 'coils_via_symmetries',
```

Comment:
ScaledCurrent should not be user exposed. It acts as a proxy class for multiplication. Please revert back this change.